test/TPP/transform/transform-convolutions.mlir

```diff
@@ -285,3 +285,145 @@ transform.sequence failures(propagate) {
     %9 = transform.structured.interchange %8 { iterator_interchange = [0, 1, 4, 2, 3, 5] }
     transform.structured.map_to_brgemm %9
 }
+
```

Comment:
please, drop this change. This should be attached to #190 and done in one go, not half-way on an uinrelated PR.